stable/kong/README.md

```diff
@@ -68,9 +68,17 @@ and their default values.
 | admin.ingress.path             | Ingress path.                                                                    | `/`                 |
 | admin.ingress.annotations      | Ingress annotations. See documentation for your ingress controller for details   | `{}`                |
 | proxy.useTLS                   | Secure Proxy traffic                                                             | `true`              |
-| proxy.servicePort              | TCP port on which the Kong Proxy Service is exposed                              | `8443`              |
+| proxy.servicePort              | TCP port on which the Kong Proxy Service is exposed                              | `443`               |
```

Comment:
Do we need this since we have added the keys under `proxy.http` and `proxy.tls`?
Same goes for `proxy.useTLS` key.

Also, please ensure that these keys are not referenced in other templates (as they are being referenced currently).

Reply:
```
grep -R "proxy.servicePort" *
README.md:| proxy.servicePort              | TCP port on which the Kong Proxy Service is exposed                              | `8443`              |
templates/NOTES.txt:     PORT={{ .Values.proxy.servicePort }}
templates/NOTES.txt:     kubectl port-forward --namespace {{ .Release.Namespace }} $POD_NAME {{ .Values.proxy.servicePort }}:{{ .Values.proxy.servicePort }}
templates/service-kong-proxy.yaml:    port: {{ .Values.proxy.servicePort }}
templates/ingress-proxy.yaml:{{- $servicePort := .Values.proxy.servicePort -}}
```

Please update the `NOTES.txt` and `ingress-proxy.yaml`.

Reply:
Updated.  I definitely did not search through this as well as i thought I did.  Went back and searched all four of the old params to make sure it wasnt there.

Thanks for reviewing and going through the rounds with me.  I think i go everything now.